src/ogma/features/nodeGrouping.ts

```diff
@@ -0,0 +1,205 @@
+import {StyleRule, Transformation, Node, NodeList} from '@linkurious/ogma';
+import {ConflictValue, LkEdgeData, LkNodeData, MissingValue} from '@linkurious/rest-client';
+
+import {LKOgma} from '../index';
+import {Tools} from '../../tools/tools';
+
+export const LKE_NODE_GROUPING_EDGE = 'LKE_NODE_GROUPING_EDGE';
+
+export class NodeGroupingTransformation {
+  public transformation?: Transformation<LkNodeData, LkEdgeData>;
+  public nodeGroupingStyleRule?: StyleRule<LkNodeData, LkEdgeData>;
```

Comment:
why public

Reply:
was needed to refresh the style from the frontend. I removed it as I think it is no longer needed

Reply:
I think it's still there. If you make it private, can you add '_'

Reply:
remove it